packages/lib/gpcircuits/scripts/gen-circuit-parameters.ts

```diff
@@ -26,6 +26,15 @@ const CIRCUIT_PARAMETERS = [
     maxTuples: 2,
     tupleArity: 2
   },
+  {
```

Comment:
Glad you figured out the zero cases!
As you're adding these circuits with zeroes, I'd suggest getting rid of some of the others to keep this list from growing too large.  We want to make sure important combinations are tested (large vs. small, zero vs. nonzero lists/tuples) but don't need every combination.